stumpy/mstump.py

```diff
@@ -144,7 +147,9 @@ def _multi_mass(Q, T, m, M_T, Σ_T, μ_Q, σ_Q):
         if np.isinf(μ_Q[i]):
             D[i, :] = np.inf
         else:
-            D[i, :] = core.mass(Q[i], T[i], M_T[i], Σ_T[i])
+            D[i, :] = core.mass(
```

Comment:
I disagree. While `core.mass` can eventually do the same thing when `np.isinf(μ_Q[i])`, this would do far less work than all of the pre-processing that is happening in `core.mass`. We short circuit things a lot sooner here. I would keep the `if/else` as-is

Reply:
Got your point! Thanks for the explanation!